macos/FluentUI/Button.swift

```diff
@@ -53,6 +66,7 @@ open class Button: NSButton {
 		// Do common initialization work
 		isBordered = false
 		wantsLayer = true
+		layer?.contentsScale = window?.backingScaleFactor ?? 1.0
```

Comment:
Not sure if I * need * to do this (it worked fine before), but we do it in Avatar View and DatePicker

Reply:
do we need noimage and imageonly enum? isn't that kind of obvious looking at if there is an image or if there is a title?

Reply:
User could set an empty image, and also the imagePosition to .imageOnly. I figured we had to support that, as that is how images in buttons work on macOS in my experience

Reply:
then we should have a convenience method of init that client just needs to pass in title and not have to pass in empty nsimage like NSButton initialzie. Also, it seems weird that we are foceing imageposition in intialization. i assume nsbutton, client can change imageposition even after it is created so we should be flexible here too?

Reply:
Line 25 has an init method with no image 
`@objc public init(title: String, style: ButtonStyle = .primaryFilled)`

A common bug I always ran into with NSButton is "Why is my image on top of my text?? or "where did my image go?" after I set an image but didn't set imagePosition. But since that is the expected API, I can remove it and whatever apple defaults to, we can follow

Reply:
my bad i still get confused with github coderview gui.

Reply:
Right, so NSButton defaults to imagePosition .imageOnly after you set an image. Because of the order of operations in our init, this means that our initializer with both a title and an image only shows the image. 

That + my desire to initialize buttons in one line in the test app view controller is why I went with adding it as a param.  I would like to force a new default (.imageLeading) for this one initializer, but I can remove the imagePosition param and rewrite the test app to match